### ISSUE TITLE
New Version 3.2.1

### ISSUE THREAD

**Issue by its-rohith** [google/android-auto-companion-android#7]
The existing source code is built upon version 3.1.2. Upon inspecting the [Maven library](https://maven.google.com/web/index.html#com.google.android.car.connectionservice:connectionservice) , I discovered that a more recent version, 3.2.1, is now accessible. When can we anticipate the availability of the updated source code for this new version?

**Comment by yaoyx108** [comment]
Hi,

tl;dr: Fixed. Also I have questions.

I have pushed release 3.2.1. Please take a look. 

https://github.com/google/android-auto-companion-android/releases/tag/3.2.1

Please note that we decided to drop support for building from source code - instead, we strongly recommend pulling the SDK from google maven, like you linked.

Could you help me understand your use case? Specifically, do you work in the auto industry (if so, which company)? How do you use the companion SDK? Do you need to build from source code, and why?

Understanding these questions help me prioritize the work on GitHub. 

Thank you.
